tests/functional/pyocf/types/core.py

```diff
@@ -178,6 +178,18 @@ def set_seq_cut_off_threshold(self, threshold):
 
         self.cache.write_unlock()
 
+    def set_seq_cut_off_promotion(self, count):
+        self.cache.write_lock()
+
+        status = self.cache.owner.lib.ocf_mngt_core_set_seq_cutoff_promotion_count(
+            self.handle, count
+        )
+        if status:
+            self.cache.write_unlock()
+            raise OcfError("Error setting core seq cut off policy promotion threshold", status)
+
+        self.cache.write_unlock()
```

Comment:
Maybe put `self.cache.write_unlock()` before checking status? Just like in the previous function.